Generate/general/xslt/generateTestScript.xsl

```diff
@@ -161,39 +184,48 @@
         </xsl:element>
     </xsl:template>
     
-    <!--Add the accept format to TestScript id-->
+    <!-- Add the format for requests to the TestScript id, if specified -->
     <xsl:template match="f:TestScript/f:id/@value" mode="filter">
-        <xsl:param name="accept" tunnel="yes"/>
+        <xsl:param name="scenario" tunnel="yes"/>
+        <xsl:param name="expectedResponseFormat" tunnel="yes"/>
         <xsl:attribute name="value">
             <xsl:value-of select="."/>
-            <xsl:if test="not(ends-with(.,'-xml'))">
+            <xsl:if test="$scenario='server' and not(ends-with(.,'xml')) and not(ends-with(.,'json'))">

```

Comment:
Als je deze check doet, zou je die dan niet ook andersom moeten doen? Dus: als de id eindigt op 'json', moet accept dan niet ook json worden?

Reply:
Ja deze check is inderdaad verre van ideaal bedacht ik me al dit weekend.

Eigenlijk moet hij checken of er (naast scenario=server) or er al een accept-element aanwezig is. Zo ja: alles met rust laten.

Reply:
Nu heb je namelijk ook kans dat er wel al een accept-element aanwezig is, maar dat hij alsnog iets met de id doet 'omdat die niet aan dit speciefieke format voldoet'.